hr_skill/hr_employee_model.py

```diff
@@ -1,7 +1,6 @@
-# -*- encoding: utf-8 -*-
```

Comment:
PLZ rename and move the file according to the Odoo guidelines https://www.odoo.com/documentation/8.0/reference/guidelines.html
hr_skill/hr_employee_model.py -> hr_skill/models/hr_employee.py